api/app/tests/fireweather_bot/test_hourly_actuals.py

```diff
@@ -62,10 +64,10 @@ def build_mock_wfwx_response(station_id: str) -> MockWFWXHourlyResponse:
     future_station_codes = asyncio.Future()
     future_station_codes.set_result([station_1, station_2])
 
-    mocker.patch('app.wildfire_one.wfwx_station_list_mapper', return_value=future_station_codes)
-    mocker.patch('app.wildfire_one.get_hourly_actuals_all_stations',
+    mocker.patch('app.wildfire_one.wfwx_api.wfwx_station_list_mapper', return_value=future_station_codes)
+    mocker.patch('app.wildfire_one.wfwx_api.get_hourly_actuals_all_stations',
```

Comment:
```suggestion
    mocker.patch('wfwx_api.get_hourly_actuals_all_stations',
```

Reply:
Minor: should be able to cut down these a little bit since you're already importing `wfwx_api`